### PR TITLE
Sell Pokemons

### LINKED ISSUE #51: Update PokeList
**Is your feature request related to a problem? Please describe.**
When a user clicks the sell button, we want the sold pokemon to be taken out of the pokeList or the quantity to be reduced. 

**Describe the solution you'd like**
Write a function to handle the behavior outlines above.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
_List Changes Introduced by this PR_
1. Allows users to sell Pokemons
2. Updates the quantities and value state

## Purpose
Give the user the ability to sell Pokemons to make more money

## Approach
Wrote a function to filter our pokemon from the inventory when the user sells it. 

Closes #51

### CODE REVIEW

**Review by JorgeLVilla — APPROVED** [review]
Excited to see it in action!  Looks good to me.

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -56,6 +55,27 @@ const PokemonCard = ({
...
+        tempPokeList[pokeIndex].quantity--;
+      } else if (tempPokeList[pokeIndex].quantity === 1) {
+        tempPokeList[pokeIndex].quantity--;
+        tempPokeList = tempPokeList.filter(
+          (pokemon) => pokemon.id !== parseInt(e.target.value)
+        );
+        setMyPokeList(tempPokeList);
+      }
+    }
+  };
```
Comment (BlackCubes) [inline pokeapp/src/components/PokemonCard.js:77]: It looks like it works!!! (me thinks................)

File: pokeapp/src/components/PokemonCard.js [hunk 2]
```diff
@@ -105,6 +127,7 @@ const PokemonCard = ({
         <h3>#{id}</h3>
         {pokemon.quantity > 0 ? <p>{pokemon.quantity}</p> : ""}
       </div>
+      <button onClick={() => console.log(myPokeList)}> mypokeList</button>
```
Comment (connielion) [inline pokeapp/src/components/PokemonCard.js:130]: Looks good. Just wondering if you still need this log here. Keep up the good work!

**Review by connielion — APPROVED** [review]
Looks good!

**Review by BlackCubes — APPROVED** [review]
This is the most beautiful thing I have ever seen. Like, my man! Einstein here!
waow

**Review by Lap343 — APPROVED** [review]
Looks god except for the console.log unless if its setting up for the next issue.

File: pokeapp/src/components/PokemonCard.js (hunk 1 above)
Comment (krferrell) [reply pokeapp/src/components/PokemonCard.js:77]: It does!

File: pokeapp/src/components/PokemonCard.js (hunk 2 above)
Comment (krferrell) [reply pokeapp/src/components/PokemonCard.js:130]: I was using it to test the functionality. I will remove it. Thanks for catching it!

**Pull request merged**